src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimCallbackHandler.java

```diff
@@ -809,4 +850,15 @@ private void handleCourtLocationForRespondent2DQ(CaseData caseData, Respondent2D
     private Optional<CaseLocationCivil> buildWithMatching(LocationRefData courtLocation) {
         return Optional.ofNullable(courtLocation).map(LocationHelper::buildCaseLocation);
     }
+
+    private String defendantFlag(CaseData caseData, UserInfo userInfo) {
```

Comment:
For context, after this RespondToClaimCallbackHandler handler, a camunda task is kicked off which will generate a document (directions questionaire), this will be created for each respondent if the case is 1v2 with different solicitor, with each having their own directions questionnaire.

We need to sort this into folder based on it being defendant one or two, we would normally determine this using 

coreCaseUserService.userHasCaseRole(caseData.getCcdCaseReference()
                                                    .toString(), userInfo.getUid(), RESPONDENTSOLICITORTWO))
                                                    
But as it is a  camunda handler generating the document, the user is a system user, therefore we set   defendantFlag appropriately during RespondToClaimCallbackHandler  which we can then query in  DirectionsQuestionnaireGenerator to determine what user "generated" the document and which folder to put the document into.